### PR TITLE
Feature/mocked start

### PR DESCRIPTION
**Pull request by ericsson49**
Support for loading beacon state files added (SSZ format).

### CODE REVIEW

File: start/node/src/main/java/org/ethereum/beacon/node/Node.java [hunk 1]
```diff
@@ -109,6 +109,44 @@
...
+  private String startMode;
+
+  @CommandLine.Option(
+      names = "--force-db-clean",
+      paramLabel = "force-db-clean",
+      description = {
+          "When an initial-state is specified, but db is not empty",
+          "specifies how to resolve the problem:",
+          "  force-db-clean=true - tries to clean db",
+          "  force-db-clean=true - exits with failure status.",
```
Comment (mkalinin) [inline start/node/src/main/java/org/ethereum/beacon/node/Node.java]: There should be `false` here, shouldn't it?

File: start/common/src/main/java/org/ethereum/beacon/start/common/util/StorageUtils.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.ethereum.beacon.consensus.BeaconChainSpec;
+import org.ethereum.beacon.consensus.BeaconStateEx;
+import org.ethereum.beacon.core.BeaconBlock;
+import org.ethereum.beacon.core.state.Checkpoint;
+import tech.pegasys.artemis.ethereum.core.Hash32;
+
+/**
+ * Utility functions to initialize storage from an initial state.
+ */
+public class StorageUtils {
```
Comment (mkalinin) [inline start/common/src/main/java/org/ethereum/beacon/start/common/util/StorageUtils.java]: We might want to make this util a part of `chain` module to make it self-sufficient in terms of chain initialization.

**Review by mkalinin — APPROVED** [review]
Looks good in general. A couple of things to fix and it could be merged.

File: start/node/src/main/java/org/ethereum/beacon/node/Node.java (hunk 1 above)
Comment (ericsson49) [reply start/node/src/main/java/org/ethereum/beacon/node/Node.java]: Definitely!

File: start/common/src/main/java/org/ethereum/beacon/start/common/util/StorageUtils.java (hunk 2 above)
Comment (ericsson49) [reply start/common/src/main/java/org/ethereum/beacon/start/common/util/StorageUtils.java]: Yes. This also will help fixing failed tests.

**Pull request merged**